viewing_party/party.py

```diff
@@ -1,23 +1,276 @@
 # ------------- WAVE 1 --------------------
+from collections import Counter
+from enum import unique
 
+# inputs a movie into the program
 def create_movie(title, genre, rating):
-    pass
+    if title and genre and rating:

```

Comment:
Great job with this function! Couple of things to note.

You could convey intent a little more explicitly by pulling "return None" into it's own guard clause with (on line 7 so it's the first line of code that executes in this function):
```python
if not title or not genre or not rating:
      return None
```
then proceed to create your movie dict

This way, when another developer comes along, they can immediately recognize "Oh if these values aren't valid, then we'll return None. The way your code is currently structured, I can tell at some point this method will return None, but it's not as immediately clear.

Also a note about guard clauses - The Guard Clause pattern helps in these cases by “inverting” that way of thinking and rapidly returning false conditions and keeping the important code blocks (usually the longest) in the lowest indentation level possible.